systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -617,10 +617,14 @@ Deployment getDeploymentFromYaml(String yamlPath) {
     }
 
     DoneableDeployment clusterOperator(String namespace) {
-        return clusterOperator(defaultCLusterOperator(namespace).build());
+        return clusterOperator(defaultCLusterOperator(namespace, "300000").build());
```

Comment:
Wouldn't it be cleaner if this just calls the other method? Or did I missed some difference between them?

```suggestion
        return clusterOperator(namespace, "300000");
```

Reply:
Definitely yes